lib/src/models/wallet.dart

```diff
@@ -52,6 +52,8 @@ class Wallet {
 
   Future<String> getOwner() async => encodeBytesToBase64(
       await _keyPair!.extractPublicKey().then((res) => res.n));
+  Future<RsaPublicKey> getPublicKey() async =>
+      await _keyPair!.extractPublicKey();
```

Comment:
```suggestion

  Future<RsaPublicKey> getPublicKey() async =>
      await _keyPair!.extractPublicKey();

```
Add empty lines before and after the block to keep consistency.